python/ray/dashboard/datacenter.py

```diff
@@ -113,7 +112,9 @@ def merge_workers_for_node(cls, node_physical_stats, node_stats):
         for worker in node_physical_stats.get("workers", []):
             worker = dict(worker)
             pid = worker["pid"]
-            worker["coreWorkerStats"] = pid_to_worker_stats.get(pid, {})
+            core_worker_stats = pid_to_worker_stats.get(pid)
```

Comment:
this is not exactly a revert?

the original code was more like:

`pid_to_worker_stats.get(pid, [])`

are we sure this is the right change?

Reply:
Yes.